src/bootstrap/compile.rs

```diff
@@ -486,7 +488,8 @@ impl Step for Rustc {
         println!("Building stage{} compiler artifacts ({} -> {})",
                  compiler.stage, &compiler.host, target);
 
-        let stage_out = builder.stage_out(compiler, Mode::Librustc);
+        let stage_out = builder.stage_out(compiler, Mode::Librustc)
```

Comment:
hm, instead of joining on target we probably want to call `cargo_out` here instead. Does that seem reasonable?